libs/s25client/s25client.cpp

```diff
@@ -570,12 +570,12 @@ int main(int argc, char** argv)
         result = e.code;
     } catch(const std::exception& e)
     {
-        bnw::cerr << "An exception occurred: " << e.what() << "\n\n";
+        bnw::cerr << "An exception occurred: " << e.what() << std::endl;
```

Comment:
I though one shouldn't use `std::endl` on non-buffered outputs (e.g `cerr`)

Reply:
It is usually superflous as `std::sync` is implicit in unbuffered streams. However I noticed that it isn't for the Boost::nowide `cerr`, which is a bug I now reported.
What I noticed is that loading a broken replay (actually due to broken code) terminated the game without any output except for the crash dialog. This causes the flush.

Reason for not using `std::endl` is the degraded performance due to the superflous sync. However here we ware terminating anyway so performance doesn't matter and the potentially additional sync doesn't hurt us.

Reply:
Turns out there was a bug in Boost::nowide that has since been fixed. We still pulled in a pre-Boost 1.77 version though (in my local tests) which lead to this.

I updated the fetched version to include the fix.